src/realmd/AuthSocket.cpp

```diff
@@ -1075,23 +1069,22 @@ bool AuthSocket::_HandleXferAccept()
 {
     DEBUG_LOG("Entering _HandleXferAccept");
 
-    recv_skip(1);
+    ReadSkip(1);
 
-    InitPatch();
+    //InitPatch();
 
     return true;
 }
 
-void AuthSocket::InitPatch()
-{
-    PatchHandler* handler = new PatchHandler(ACE_OS::dup(get_handle()), patch_);
-
-    patch_ = ACE_INVALID_HANDLE;
-
-    if (handler->open() == -1)
-    {
-        handler->close();
-        close_connection();
-    }
-}
-
+//void AuthSocket::InitPatch()
```

Comment:
Yes.  It is worth mentioning that once the ASIO implementation is complete, I intend to pull realmd into its own repository.